### DISCUSSION TITLE
Build image for UEFI bios

### DISCUSSION THREAD

**Discussion by ghost** [oracle/oracle-linux#121] [General Oracle Linux questions]
Hi all,
here is the notes how to make OL build... but I have question if is possible make image with uefi supports... (i mean efi partitions and install grub2-efi)...

Thanks for information!

Stepan

**Comment by AmedeeBulle** [comment]
Hi,
The aarcch64 builds are UEFI based, but not the x86-64.
There is currently no option in the scripts to make an UEFI image, but that shouldn't be too complicated. We might consider adding an option for that.
